### ISSUE TITLE
Issue - Tuple Attribute has no 'to' 

### ISSUE THREAD

**Issue by TapanManu** [udacity/deep-learning-v2-pytorch#241]
This error is always popping up when the following snippet is  executed.
**Part-8 Transfer Learning Solution**
`epochs = 1
steps = 0
running_loss = 0
print_every = 5
for epoch in range(epochs):
    for inputs, labels in trainloader:
        steps += 1
        # Move input and label tensors to the default device
        inputs, labels = inputs.to(device), labels.to(device)
        
        optimizer.zero_grad()
        
        logps = model.forward(inputs)
        loss = criterion(logps, labels)
        loss.backward()
        optimizer.step()

        running_loss += loss.item()
        
        if steps % print_every == 0:
            test_loss = 0
            accuracy = 0
            model.eval()
            with torch.no_grad():
                for inputs, labels in testloader:
                    inputs, labels = inputs.to(device), labels.to(device)
                    logps = model.forward(inputs)
                    batch_loss = criterion(logps, labels)
                    
                    test_loss += batch_loss.item()
                    
                    # Calculate accuracy
                    ps = torch.exp(logps)
                    top_p, top_class = ps.topk(1, dim=1)
                    equals = top_class == labels.view(*top_class.shape)
                    accuracy += torch.mean(equals.type(torch.FloatTensor)).item()
                    
            print(f"Epoch {epoch+1}/{epochs}.. "
                  f"Train loss: {running_loss/print_every:.3f}.. "
                  f"Test loss: {test_loss/len(testloader):.3f}.. "
                  f"Test accuracy: {accuracy/len(testloader):.3f}")
            running_loss = 0
            model.train()`

**Comment by TapanManu** [comment]
This is the exact code available in the repository .On executing this,The Error gets popped up in the second loop for the 'testloader'.I am actually new to pytorch .I can't figure it out why such an error occurs? 
Sorry,the error is mentioned in the title.
I would really appreciate for helping me out of this.

**Comment by abhiojha8** [comment]
Though we are a little late to respond, we suggest you reach out to Udacity support for user-specific issues. 
Thanks
